### PR TITLE
Change app menu focused items behavior that neighbor elements (text and border) have no overlapping

### LINKED ISSUE #35201: [BITV] Focused item overlaps neighbor elements with text and border
> * Keyboard focus overlaps with text

@jancborchardt  you mean 

Screenshot from 2022-11-15 11-27-55 ?

Then i guess we need another design

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
Add native tooltips to items.

Implementation / new UX was discussed with @jancborchardt:

1. Increase the width of the elements a little bit.
2. Simply do not expand the text anymore and just use title. Use ellipsed name of the app.
3. Use the ::after style instead of the border around it. Should be same as for active app but thin.
4. Use letter-spacing: -0.5px for names.
5. Take "zooming" effect on focus away.

❗Accessibility improvement

Fixes https://github.com/nextcloud/server/issues/35201

Before: 

Screenshot from 2022-11-15 11-27-55 

Screenshot from 2022-11-29 18-01-43

After:

menu2

menu

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]
Looks good

File: core/src/components/AppMenu.vue [hunk 1]
```diff
@@ -254,10 +257,7 @@ $header-icon-size: 20px;
 
 	&:focus-visible {
 		opacity: 1;
-		background-color: transparent !important;
-		border-radius: var(--border-radius);
-		outline: none;
-		box-shadow: 0 0 0 2px var(--color-primary-text);
+		outline: none!important;
```
Comment (Pytal) [inline core/src/components/AppMenu.vue:260]:
```suggestion
		outline: none !important;
```

**Review by Pytal — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]

**Comment by JuliaKirschenheuter** [comment]
After some rebasing-job and after this PR has been merged i've seen that some changes from server.css weren't merged. Because of this i'm opening a second PR which will fix this styles.

**Pull request merged**